test/DynamoCoreWpfTests/PackageManagerUITests.cs

```diff
@@ -91,6 +95,39 @@ public void PackagePublishWindowClosesWithDynamo()
 
         }
 
+        [Test]
+        public void AdddingaPackageRaisesCanExecuteChangeOnDelegateCommand()
+        {
+            //create a new synchronization context since the package searchview from previous UI tests is still
+            //running and throws exception when attempting to modify searchresults
+            SynchronizationContext.SetSynchronizationContext(new SynchronizationContext());
+            var vm = new PublishPackageViewModel(ViewModel);
+            ViewModel.OnRequestPackagePublishDialog(vm);
+
+            //find a customnode to add to the package
+            string packagedirectory = Path.Combine(GetTestDirectory(ExecutingDirectory), "pkgs");
+            var packages = Directory.EnumerateDirectories(packagedirectory);
+            var first = Path.GetFullPath(packages.First());
+            string dyfpath = Path.Combine(first, "dyf");
+            var customnodes = Directory.GetFiles(dyfpath);
+            var firstnode = customnodes.First();
+                      
+            var canExecuteChangedFired = 0;
+            vm.SubmitCommand.CanExecuteChanged += ((o, e) => { canExecuteChangedFired ++; });
+            //now use reflection to add a customnode to the package
+            var propertyChangedmethod = typeof(PublishPackageViewModel).GetMethods(BindingFlags.NonPublic | BindingFlags.Instance).Where(x=>x.Name == "AddFile").First();
+            propertyChangedmethod.Invoke(vm, BindingFlags.InvokeMethod,null, new object[] { firstnode },System.Globalization.CultureInfo.CurrentCulture);
+            
```

Comment:
@holyjewsus If this is a private method, it's fine if you simply make it `internal` and call it here.  `internal` methods are exposed to DynamoCoreWpfTests using the `InternalsVisibleTo` property.